crates/ty_python_semantic/src/types/instance.rs

```diff
@@ -45,12 +45,13 @@ impl<'db> Type<'db> {
         protocol: ProtocolInstanceType<'db>,
     ) -> bool {
         // TODO: this should consider the types of the protocol members
-        // as well as whether each member *exists* on `self`.
-        protocol
-            .inner
-            .interface(db)
-            .members(db)
-            .all(|member| !self.member(db, member.name()).symbol.is_unbound())
+        // as well as whether each member *exists fully bound* on `self`.
```

Comment:
Doesn't it already do this part? So it seems weird to have it as part of a TODO comment
```suggestion
```